src/engraving/dom/textbase.cpp

```diff
@@ -1923,28 +1927,49 @@ bool TextBase::prepareFormat(const String& token, CharFormat& format)
     } else if (token == "/sup") {
         format.setValign(VerticalAlignment::AlignNormal);
     } else if (token.startsWith(u"font ")) {
-        String remainder = token.mid(5);
-        if (remainder.startsWith(u"size=\"")) {
-            format.setFontSize(parseNumProperty(remainder.mid(6)));
-            return true;
-        } else if (remainder.startsWith(u"face=\"")) {
-            String face = parseStringProperty(remainder.mid(6));
-            face = unEscape(face);
-            format.setFontFamily(face);
-            return true;
-        } else {
-            LOGD("cannot parse html property <%s> in text <%s>", muPrintable(token), muPrintable(m_text));
+        String remainder = token.mid(5).trimmed();
+        while (!remainder.empty()) {
+            if (remainder.startsWith(u"size=\"")) {
+                double size = parseNumProperty(remainder.mid(6));
+                remainder = remainder.mid(remainder.indexOf(u'"', 6) + 1).trimmed();
+                if (!token.endsWith(u"/")) {
+                    prevFontSize = format.fontSize();
+                }
+                format.setFontSize(size);
+            } else if (remainder.startsWith(u"face=\"")) {
+                String face = unEscape(parseStringProperty(remainder.mid(6)));
+                remainder = remainder.mid(remainder.indexOf(u'"', 6) + 1).trimmed();
+                if (!token.endsWith(u"/")) {
+                    prevFontFace = format.fontFamily();
+                }
+                format.setFontFamily(face);
+            } else {
+                LOGD("cannot parse html property <%s> in text <%s>", muPrintable(remainder), muPrintable(m_text));
```

Comment:
This code seems to get hit with the `/` from `<font />`, maybe that is related to the problem

Reply:
Hmm, I believe that case needs to get handled separately and as a closing tag, returning false.

Reply:
OK, let's break earlier on that case

Reply:
That (alone) doesn't help though.
I'm at a loss to find where and why it fails